addons/account/tests/test_account_move_send.py

```diff
@@ -1081,3 +1081,24 @@ def test_no_sending_method_selected(self):
         wizard.action_send_and_print()
         self.assertTrue(invoice.is_move_sent)
         self.assertTrue(invoice.invoice_pdf_report_id)
+
+    def test_get_sending_settings(self):
+        invoice = self.init_invoice("out_invoice", amounts=[1000], post=True)
+        wizard = self.create_send_and_print(invoice)
+        
+        expected_results = {
+            'sending_methods': ['email'],
+            'invoice_edi_format':False,
+            'extra_edis': [],
+            'pdf_report': self.env['ir.actions.report'].search([('name', '=', 'PDF')]),
+            'author_user_id': self.env.user.id,
+            'author_partner_id': self.env.user.partner_id.id,
+            'mail_template': self.env['mail.template'].search([('name', '=', 'Invoice: Sending')]),
+            'mail_lang': 'en_US',
+            'mail_body': wizard.mail_body,
+            'mail_subject': 'company_1_data Invoice (Ref INV/2019/00001)',
+            'mail_partner_ids': [invoice.partner_id.id],
```

Comment:
Not asking for change but could be done like this:
```suggestion
            'mail_partner_ids': invoice.partner_id.ids,
```